core/services/relay/evm/mercury/config_poller.go

```diff
@@ -206,6 +209,11 @@ func (cp *ConfigPoller) startLogSubscription() {
 			continue
 		}
 
+		address := addressTopicValues[0]
```

Comment:
This can panic if `addressTopicValues` is empty

Reply:
There's a length check above to check that `addressTopicValues` has length at least 2. There's also unit tests that test different event string formats including empty strings, missing addresses, etc.